src/main/java/com/lms/learning_management_system/DTO/UserDTO.java

```diff
@@ -0,0 +1,20 @@
+package com.lms.learning_management_system.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+
+import java.util.UUID;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserDTO {
+    private UUID id;
+    private String firstName;
+    private String lastName;
+    private String email;
+    private String roleId;
```

Comment:
I thing its better to be "role", not "roleId"

Reply:
in the user crud branch it is simply role, will be resolved at merge as a conflict